.travis.yml

```diff
@@ -18,9 +18,9 @@ notifications:
 matrix:
   exclude:
     - python: "3.3"
-      env: DJANGO='django>=1.4,<1.5'
+      env: DJANGO='django>=1.7,<1.8'
     - python: "3.4"
-      env: DJANGO='django>=1.4,<1.5'
+      env: DJANGO='django>=1.7,<1.8'
```

Comment:
If we're dropping 1.4 support, I think the exclude block can be dropped entirely.